devtools/server/actors/replay/connection.js

```diff
@@ -29,15 +29,6 @@ let gConfig;
 // When connecting we open an initial channel for commands not associated with a recording.
 let gMainChannelId;
 
-function getLoggedInUserAuthId() {
```

Comment:
We accidentally duplicated this function, there's another definition farther down.